docs/source/markdown/options/network.md

```diff
@@ -59,6 +65,7 @@ Valid _mode_ values are:
     **-U none** are given to disable the same functionality from container to
     host. \
     Some examples:
+
```

Comment:
if you repush, im not sure this needs to be here?

Reply:
Yep, it seems to render fine without it.